packages/astro/src/@types/astro.ts

```diff
@@ -895,11 +896,40 @@ export interface AstroUserConfig {
 		astroFlavoredMarkdown?: boolean;
 	};
 
-	/**
-	 * @hidden
-	 * Turn on experimental support for the new error overlay component.
+		/**
+	 * @docs
+	 * @kind heading
+	 * @name Experimental Flags
+	 * @description
+	 * Astro offers experimental flags to give users early access to new features.
+	 * These flags are not guaranteed to be stable.
 	 */
-	experimentalErrorOverlay?: boolean;
+	experimental?: {
+		/**
+			* @hidden
+			* Turn on experimental support for the new error overlay component.
+			*/
+		errorOverlay?: boolean;
+		/**
+		 * @docs
+		 * @name experimental.prerender
+		 * @type {boolean}
+		 * @default `false`
+		 * @version 1.7.0
+		 * @description
+		 * Enable experimental support for Prerendered pages when generating a server.
```

Comment:
No need to capitalize here I don’t think.

```suggestion
		 * Enable experimental support for prerendered pages when generating a server.
```